src/desktop/apps/article/routes.ts

```diff
@@ -146,7 +146,7 @@ export async function index(req, res, next) {
     res.locals.sd.RESPONSIVE_CSS = createMediaStyle()
 
     // CollectionsRail a/b test
-    splitTest("editorial_collections_rail").view()
+    // splitTest("editorial_collections_rail").view()
     const hasCollectionsRail = EDITORIAL_COLLECTIONS_RAIL === 1
```

Comment:
If the test has been running for a few days we may already have users where an `EDITORIAL_COLLECTIONS_RAIL` cookie has been set -- we should probably set `hasCollectionsRail` to false as well.